rocky/reports/report_types/mail_report/report.py

```diff
@@ -0,0 +1,77 @@
+from dataclasses import dataclass
+from datetime import datetime
+from logging import getLogger
+from typing import Any, Dict
+
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models import Reference
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6
+from reports.report_types.definitions import Report
+
+logger = getLogger(__name__)
+
+
+@dataclass
+class System:
+    system_types: list
+    oois: list
+
+
+class MailReport(Report):
+    id = "mail-report"
+    name = _("Mail Report")
+    description = _("System specific mail report that focusses on IP addresses and hostnames.")
+    plugins = {"required": ["nmap"], "optional": []}
+    input_ooi_types = {Hostname, IPAddressV4, IPAddressV6}
+    template_path = "mail_report/report.html"
+
+    number_of_spf = 0
+    number_of_dmarc = 0
+    number_of_dkim = 0
```

Comment:
These are instance attributes, not class variables. Therefore they should be defined in `__init__`